docs/api/cameras/OrthographicCamera.html

```diff
@@ -64,7 +64,12 @@
 		<div>Camera frustum bottom plane.</div>
 
 		<h3>[property:Float far]</h3>
-		<div>Camera frustum far plane. Default is *2000*.</div>
+		<div>
+		Camera frustum far plane. Default is *2000*.<br /><br />
+
+		Note that Three does not currently give warnings for incorrect values here, however the expected
```

Comment:
I think we use "three.js", not "Three". It is not capitalized. There are multiple occurrences of this error.

Reply:
@looeee A bit late, but, no.  :)